### DISCUSSION TITLE
Using a block-enabled Full Site Editing (FSE) theme with FaustJS

### DISCUSSION THREAD

**Discussion by TMFRyan** [wpengine/faustjs#1483] [Q&A]
So I know we can use the https://github.com/wpengine/wp-graphql-content-blocks extension for WPGraphQL to query editor blocks in a post.

How do we target blocks that are defined as part of a block-enabled theme like Twenty Twenty-Two, included in various templates and template parts. 

For example, I've added the site logo to the site logo block in the header template part. How do I expose this to the app so I can render it in the header component?

**Comment by theodesp** [comment]
Hey @TMFRyan . FSE support with wp-graphql-content-blocks is not supported ATM.

However there is an experimental plugin you can check out:

https://github.com/AxeWP/wp-graphql-site-editor
